archivebox/util.py

```diff
@@ -175,7 +175,7 @@ def download_url(url: str, timeout: int=None) -> str:
 def chrome_args(**options) -> List[str]:
     """helper to build up a chrome shell command with arguments"""
 
-    from .config import CHROME_OPTIONS
+    from archivebox.config import CHROME_OPTIONS
```

Comment:
hmm I don't think this is the right solution, I think you should put `__package__ = 'archivebox'` at the top of the file instead